### ISSUE TITLE
Create idle, walk, run cycles for Bob Sample Rig

### ISSUE THREAD

**Issue by EricTRocks** [fabric-engine/Kraken#459]
To be able to create better looking samples and demos we need to have some source animations for the Bob character.

**Comment by BigRoy** [comment]
How would you want these created? Animation done in Maya for example?

**Comment by EricTRocks** [comment]
Yes and we'll probably need them exported to fbx and / or alembic
